.dotnet/src/Custom/FineTuning/FineTuningJobOperation.cs

```diff
@@ -0,0 +1,206 @@
+    /// <returns> The response returned from the service. </returns>
+    public virtual async Task<ClientResult<FineTuningJob>> GetJobAsync(CancellationToken cancellationToken = default)
+    {
+        using PipelineMessage message = CreateRetrieveFineTuningJobRequest(_jobId, cancellationToken.ToRequestOptions());
+        return (ClientResult<FineTuningJob>)ClientResult.FromResponse(await _pipeline.ProcessMessageAsync(message, cancellationToken.ToRequestOptions()).ConfigureAwait(false));
+    }
+
+    /// <summary>
+    /// Get info about a fine-tuning job.
+    ///
+    /// [Learn more about fine-tuning](/docs/guides/fine-tuning)
+    /// </summary>
+    /// <param name="cancellationToken"> The cancellation token to use. </param>"
+    /// <exception cref="ClientResultException"> Service returned a non-success status code. </exception>
+    /// <returns> The response returned from the service. </returns>
+    public virtual ClientResult<FineTuningJob> GetJob(CancellationToken cancellationToken = default)
+    {
+        using PipelineMessage message = CreateRetrieveFineTuningJobRequest(_jobId, cancellationToken.ToRequestOptions());
+        var rawResponse = _pipeline.ProcessMessage(message, cancellationToken.ToRequestOptions());
+        return ClientResult.FromValue(FineTuningJob.FromResponse(rawResponse), rawResponse);
+    }
+
+    /// <summary>
+    /// Immediately cancel a fine-tune job.
+    /// </summary>
+    /// <param name="cancellationToken"> The cancellation token to use. </param>
+    /// <exception cref="ClientResultException"> Service returned a non-success status code. </exception>
+    /// <returns> The response returned from the service. </returns>
+    public virtual async Task<ClientResult<FineTuningJob>> CancelAsync(CancellationToken cancellationToken = default)
+    {
+        using PipelineMessage message = CreateCancelFineTuningJobRequest(_jobId, cancellationToken.ToRequestOptions());
+        var rawResponse = await _pipeline.ProcessMessageAsync(message, cancellationToken.ToRequestOptions()).ConfigureAwait(false);
+        return ClientResult.FromValue(FineTuningJob.FromResponse(rawResponse), rawResponse);
+    }
+
+    /// <summary>
+    /// Immediately cancel a fine-tune job.
+    /// </summary>
+    /// <param name="cancellationToken"> The cancellation token to use. </param>
+    /// <exception cref="ClientResultException"> Service returned a non-success status code. </exception>
+    /// <returns> The response returned from the service. </returns>
+    public virtual ClientResult<FineTuningJob> Cancel(CancellationToken cancellationToken = default)
+    {
+        var method = CreateCancelFineTuningJobRequest;
+        PipelineMessage message;
+        return NewMethod(method, out message, cancellationToken);
+    }
+
+    private ClientResult<FineTuningJob> NewMethod(Func<string, RequestOptions?, PipelineMessage> method, out PipelineMessage message, CancellationToken cancellationToken)
```

Comment:
What does this do? Maybe a comment to describe it?

Reply:
Whoa, looks like an unintentional refactor. Removed.